content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -15,6 +15,76 @@ This document serves both as a reference to the values and as a coordination poi
 
 ## Labels, annotations and taints used on API objects
 
+### app.kubernetes.io/component
+
+Example: `app.kubernetes.io/component=database`
+
+Used on: All Objects
```

Comment:
unsure if this should be `All Objects` or if we need to specifically list the objects since these labels may not be applicable to Nodes?

Reply:
How about adding a text note that this is usually used as a label on a [workload object](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/), or on something that an workload object directly depends on.